client/src/pages/AddReview/style.css

```diff
@@ -0,0 +1,24 @@
+.header-img {
+  background-size: cover;
+  background-image: url("https://images.unsplash.com/photo-1536007164800-b7f11331f35c?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1950&q=80");
```

Comment:
you can leave it for your purposes, just be aware that images from 3rd parties are outside of your control and could be removed any time - you should have any images used by your app in your repo..

Reply:
good point, noted